src/lib/crypto.h

```diff
@@ -63,6 +63,7 @@
 #include "packet-parse.h"
 #include "symmetric.h"
 #include "bn.h"
+#include <rekey/rnp_key_store.h>
```

Comment:
`crypto.h` uses `rnp_key_store_t`, the only reason that include wasn't in there before is because it was indirectly pulled in from `packet.h` (which included it for `pgp_parse_and_accumulate`, which I removed).
:)